WindowsServerDocs/get-started-19/activation-19.md

```diff
@@ -13,6 +13,15 @@ ms.date: 09/09/2020
 
 >Applies To: Windows Server 2019, Windows Server 2016
 
+> [!NOTE]
+> It is greatly advised to implement ADBA (Active Directory Activation) instead of KMS, aswell as migrating existing KMS for modern products to ADBA. 
+> ADBA supports every OS of Windows Server 2012 and newer, respectively Windows 8.1 or newer, aswell as Office 2013 or newer.
+> An exception are Microsoft 365 Office Apps for Business, which are activated via Internet, only.
+>
+> References: 
+> [How to implement ADBA](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/activate-using-active-directory-based-activation-client)
+> [ADBA Overview & Scenarios](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/active-directory-based-activation-overview) 
```

Comment:
Grammar improvement suggestions and forced line breaks:
```suggestion
> [!NOTE]
> It is highly recommended to implement ADBA (Active Directory Activation) instead of KMS, as well as migrating existing KMS for modern products to ADBA.<br/>
> ADBA supports Windows Server 2012 and newer, Windows 8.1 or newer, as well as Office 2013 or newer.<br/>
> One exception is Microsoft 365 Office Apps for Business, which can only be activated via the Internet.
>
> References:<br/>
> [How to implement ADBA](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/activate-using-active-directory-based-activation-client)<br/>
> [ADBA Overview & Scenarios](https://docs.microsoft.com/en-us/windows/deployment/volume-activation/active-directory-based-activation-overview)
```